Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM debian:stretch-slim
+MAINTAINER SoloKeys <hello@solokeys.com>
+
+# get build requirements
+RUN apt-get update -qq
+RUN apt-get install -qq git make wget bzip2 >/dev/null
+
+# get Python3.7
+RUN wget -q https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh
+RUN bash ./Miniconda3-latest-Linux-x86_64.sh -b -p /tmp/conda
+RUN ln -s /tmp/conda/bin/python3 /usr/local/bin/python3
```

Comment:
1. I couldn't find sha256sums for ARM GCC, but I guess I could add my own from an initial download. Agreed on adding a pinning + sha256sum for Conda.
2. I'd like to be using Python3.7, or at least 3.6 for f-string support... We did recently decide to definitely drop Python2 support (it's EOL this year). Debian stable and Ubuntu Xenial are at 3.5 I believe. I'd rather not use deadsnakes PPA for Debian compatibility. Maybe just swallow the pill and use Ubuntu Bionic?
3. It's used to merge `solo.hex` and `bootloader.hex`. I think it would be helpful to have official and nightly builds to include `all.hex`. Good idea to do this in a separate stage.

Will iterate a bit, thanks for the input!

Reply:
Haha, `black` only supports Python 3.6+ :)